test_util/aws.py

```diff
@@ -115,3 +125,116 @@ def get_public_agent_ips(self):
     def get_private_agent_ips(self):
         instances = self.get_tag_instances('SlaveServerGroup')
         return instances_to_hosts(instances)
+
+
+class VpcCfStack(CfStack):
+    @classmethod
+    def create(cls, stack_name, instance_type, instance_os, instance_count,
+               admin_location, key_pair_name, boto_wrapper):
+        ami_code = OS_AMIS[instance_os][boto_wrapper.region]
+        template_url = template_by_instance_type(instance_type)
+        parameters = {
+            'KeyPair': key_pair_name,
+            'AllowAccessFrom': admin_location,
+            'ClusterSize': str(instance_count),
+            'InstanceType': str(instance_type),
+            'AmiCode': ami_code}
+        stack = boto_wrapper.create_stack(stack_name, template_url, parameters)
+        return cls(stack.stack.stack_id, boto_wrapper)
+
+    def delete(self):
+        # boto stacks become unusable after deletion (e.g. status/info checks) if name-based
+        self.stack = self.boto_wrapper.resource('cloudformation').Stack(self.stack.stack_id)
+        self.stack.delete()
+
+    def get_vpc_host_ips(self):
+        reservations = self.boto_wrapper.client('ec2').describe_instances(Filters=[{
+            'Name': 'tag-value', 'Values': [self.stack.stack_name]}])['Reservations']
+        logging.debug('Reservations for {}: {}'.format(self.stack.stack_id, reservations))
+        instances = reservations[0]['Instances']
+        return instances_to_hosts(instances)
+
+
+OS_AMIS = {
+    'cent-os-6': {'ap-northeast-1': 'ami-82640282',
```

Comment:
Was copy/pasting from CCM so that anyone could expect the same set of AMIs in this API, but yes, definitely pointless to keep around if its a DC/OS specific utility